05-Organizing-and-Executing-Simulations/04-Submitting-Cluster-Jobs.ipynb

```diff
@@ -0,0 +1,611 @@
+{
```

Comment:
<p>Same for the first bullet point here. signac-flow is both partitioning and aggregating?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Aggregating is more general than partitioning. signac-flow provides the aggregation and here we are using that to run the jobs in separate MPI partitions. </p><p> </p><p>In the example I mentioned above, you could write an aggregate operation to instead loop over all the jobs and average some quantity. Such an operation would most likely request only rank and not <code>RANKS_PER_PARTITION * len(jobs).</code></p>